proto/api.go

```diff
@@ -79,6 +79,7 @@ type Request interface {
 	gogoproto.Message
 	// Header returns the request header.
 	Header() *RequestHeader
+	KVHeader() *KVRequestHeader
```

Comment:
If `Call` is still going to be used everywhere, the `Request` and `Response` it contains cannot be KV specific or you haven't bought anything with this change.